netlify.toml

```diff
@@ -14,5 +14,5 @@ for = "/dist/client/*"
 # These are only used to set up the template, and are not used in the build
 # If you want to update the real values, change them in the site UI or CLI
 [template.environment]
-PUBLIC_STORE_DOMAIN = "Store domain. Leave as 'mock.shop' to try the demo site"
+PUBLIC_STORE_DOMAIN = "Store domain - set to 'mock.shop' to try the demo site"
 SESSION_SECRET = "Session secret - change to a random value for production"
```

Comment:
Should the similar wording change be applied to `SESSION_SECRET` as well if we are not pre-populating it with anything?

Reply:
> if we are not pre-populating it with anything

Well, it's the opposite really. We're _always_ pre-populating it with the string "Session secret - change to a random value for production". This populates a _default_ - there isn't really a feature to provide a "description". We're using the default to serve as a description/instruction.

So, I guess if you ignored this instruction, your `SESSION_SECRET` would be the string "Session secret - change to a random value for production". 🤷🏼

I'll write up a feature request to support separate fields for defaults vs. descriptions...

Reply:
Oh, wait. I was confused.
<img width="654" alt="Screenshot 2024-09-09 at 11 44 20" src="https://github.com/user-attachments/assets/0236d8f3-ef63-468d-aaf7-ab2a43a8462a">